src/main/java/net/masterthought/cucumber/generators/OverviewReport.java

```diff
@@ -115,4 +122,29 @@ public String getName() {
     public Status getStatus() {
         throw new NotImplementedException();
     }
+
+    @Override
+    public FeatureScenario[] getFeatureDetails() {
+        //get all feature & scenario name
+        List<Feature> features = Features.getFeatures();
+        FeatureScenario[] list_tfs = new FeatureScenario[0];
+
+        for (Feature feature : features){
+            String featureName = feature.getName();
+            String deviceName = feature.getDeviceName();
+            Element[] scenarios = feature.getElements();
+            for (Element scenario : scenarios){
+                if(scenario.isScenario()){
+                    String scenarioName = scenario.getName();
+                    String scenarioStatus = scenario.getStatus().toString();
+                    String line = scenario.getLine();
+
+                    FeatureScenario trendFeatureScenario = new FeatureScenario(deviceName, featureName, scenarioName, scenarioStatus,line);
+                    list_tfs = (FeatureScenario[]) ArrayUtils.add(list_tfs, trendFeatureScenario);
```

Comment:
`list_fts` is not Java naming convention

Reply:
done